facebook/__init__.py

```diff
@@ -82,7 +82,7 @@ class GraphAPI(object):
 
     """
 
-    def __init__(self, access_token=None, timeout=None, version=None):
+    def __init__(self, access_token=None, timeout=None, version=None, **kwargs):
```

Comment:
Why `**kwargs` here instead of `app_secret=None`?

Also, this line is [too long](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).